lib/billy/railtie.rb

```diff
@@ -1,9 +1,7 @@
-module Billy
-  class Railtie < Rails::Railtie
-    railtie_name 'billy'
+# Deprecated
+require 'billy/browsers/capybara'
+require 'billy/init/railtie'
 
-    rake_tasks do
-      load 'tasks/billy.rake'
-    end
-  end
-end
+Billy::Browsers::Capybara.register_drivers
+
+warn "[DEPRECATION] `require 'billy/railtie'` is deprecated. Please use `require 'billy/capybara/railtie'` instead."
```

Comment:
By making this generic again, we won't need any deprecation warning for this one.  In fact, I don't think it should ever be required separately as it just hooks into Rails on load so no warning needed either way.